gradle.properties

```diff
@@ -14,26 +14,15 @@
 # contact D4L by email to help@data4life.care.
 #
 
-# Project-wide Gradle settings.
-
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
-
-android.enableJetifier=true
-android.useAndroidX=true
-
-org.gradle.configureondemand=false
-
+# Gradle
 org.gradle.jvmargs=-Xmx1536m
+org.gradle.parallel=false
```

Comment:
Why we don't run builds not any longer in parallel?

Reply:
it is flaky due to race conditions